analyzer/QUIC.spicy

```diff
@@ -243,7 +243,18 @@ type InitialPacket = unit(header: LongHeaderPacket) {
   token_length: VariableLengthInteger;
   token: bytes &size=self.token_length.result;
   length: VariableLengthInteger;
-  # Consume the remainder of payload. This is
+
+  # 5.4.2. Header Protection Sample
+  #
+  #   That is, in sampling packet ciphertext for header
+  #   protection, the Packet Number field is assumed to
+  #   be 4 bytes long (its maximum possible encoded length).
+  #
+  # Enforce 4 bytes Packet Number length + 16 bytes sample
+  # ciphertext available.
+  : bytes &size=0 &requires=self.length.result >= 20;
```

Comment:
A more explicit version would be
```ruby
  : void &requires=self.length.result >= 20;
```

That said, I think it would be more natural to directly attach this to `length` since this is just validation for exactly that field.